### PR TITLE
[DEV APPROVED] add opt_in_for_research field to User record

### PR DESCRIPTION
**Pull request by Tr4pSt3R**
Depends on the successful merge of the PR for the Cream library: https://github.com/moneyadviceservice/cream/pull/15

### CODE REVIEW

File: db/migrate/20150827092935_add_opt_in_field_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddOptInFieldToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :opt_in_for_research, :boolean, default: true
```
Comment (asmega) [inline db/migrate/20150827092935_add_opt_in_field_to_users.rb]: is the default suppose to be true? 
and if the user unticks does it save the value as false?

Comment (Tr4pSt3R) [inline db/migrate/20150827092935_add_opt_in_field_to_users.rb]: @asmega yeah the default, i have been told, is to be set to true. The user will have to opt-out by unchecking the box. Yikes! haha

**Comment by darokel** [comment]
All looks good but to avoid checking in and reverting stuff, is this ready to be verified on QA and shipped?

**Comment by asmega** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:

**Comment by Tr4pSt3R** [comment]
@tomas-stefano and @asmega would you mind reviewing this again please?

**Comment by benbarnett** [comment]
@Tr4pSt3R if the PR has changed then this is no longer [DEV APPROVED] and should go back in the PR notifier ;)

**Comment by asmega** [comment]
:+1:

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**